substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -605,7 +605,28 @@ fn already_charged(_: u32) -> Option<RuntimeCosts> {
 fn extract_hi_lo(reg: u64) -> (u32, u32) {
 	((reg >> 32) as u32, reg as u32)
 }
+/// Provides storage variants to support standard and Etheruem compatible semantics.
```

Comment:
```suggestion
}

/// Provides storage variants to support standard and Etheruem compatible semantics.
```

Nit: Having extra blank line between the variants of this enum but not between completely unrelated surrounding things hinders readability.